src/mbgl/layout/symbol_layout.cpp

```diff
@@ -90,7 +92,7 @@ SymbolLayout::SymbolLayout(std::string bucketName_,
                 u8string = platform::lowercase(u8string);
             }
 
-            ft.text = util::utf8_to_utf32::convert(u8string);
+            ft.text = bidi.bidiTransform( util::utf8_to_utf32::convert( u8string ) );
```

Comment:
Since `getGlyphRange` doesn't support anything beyond the Basic Multilingual Plane, are we better off using utf16 throughout this code, rather than going utf8 ⇢ utf32 ⇢ utf16 ⇢ utf32?